packages/legacy/core/App/localization/fr/index.ts

```diff
@@ -362,29 +381,32 @@ const translation = {
         "NotAvailableInYourWallet": "Non disponible dans votre portefeuille",
         "IsRequesting": "demande",
         "IsRequestingSomethingYouDontHaveAvailable": "demande quelque chose que vous n'avez pas à votre disposition",
-        "IsRequestingYouToShare": "is requesting the following information from",
-        "Credential": "credential.",
-        "Credentials": "credentials.",
+        "IsRequestingYouToShare": "demande les informations suivantes à ",
```

Comment:
Would this be phrased better as: "demande les informations suivantes à partir de" considering the context. The full translated phrase is split in "verifier" "IsRequestingYouToShare" "number of credentials" "credential/credentials" so an example would be "ABC demande les informations suivantes à partir de 3 justificatifs d'identité."